dist/mapdc.js

```diff
@@ -84442,8 +84453,6 @@ function mapdTable(parent, chartGroup) {
     } else if (type === "DATE") {
       var dateFormat = _d2.default.time.format.utc("%Y-%m-%d");
       val = "DATE '" + dateFormat(val) + "'";
-    } else if (val && typeof val === "string") {
-      val = "'" + val.replace(/'/g, "''") + "'";
```

Comment:
This was mapping `"foo"` -> `"'foo'"`. I have to assume this was done for sql building purposes and escaping things. We don't need it any more.